### DISCUSSION TITLE
spark-doris-connector不支持分区

### DISCUSSION THREAD

**Discussion by young138120** [apache/doris#10796] [A - General / Q&A] [answered]
connector本质还是通过stream load的方式进行数据的导入  
文档中stream load是再header中添加partition的标识，但是connector的DroisStreamLoad这个实现类没有实现相关的配置

**Comment by young138120** [comment]
spark-doris-connector-3.1_2.12:1.0.1

**Comment by hf200012** [comment, marked as answer]
To import data, you do not need to specify a partition. When importing data, it will be automatically imported into the corresponding partition according to your partition fields.
